//package core.io.protocol;

message DceBattleOpt
{
	optional string	battleid	= 1;//操作的战斗ID
	optional int32	opt_type 	= 2;//1准备好 2播放一个action 3释放滑动技能 4释放怒气技能 5切换手动自动(param自动发1手动发0) 6跳过
	optional string id 			= 3;//技能释放式神ID
	optional int32	param		= 4;//怒气技能的打p效果 1 2 3 4 
	optional int32	transparam	= 5;//无逻辑 转发客户端参数
}
